Add Nordic Summer Showdown and update osu! originals (#12066)
ppy/osu-wiki: wiki/Tournaments/NSS/1/en.md

17. If players do not join the match at the given time, the following penalties apply:
    - 5 minutes: No warmup
    - 8 minutes: No protect
    - 10 minutes: No bans
    - 15 minutes: Forfeit
18. If a player becomes silenced during the match:
    - 5 minutes: Timeouts lost or next pick/ban lost
    - 10 minutes: Timeouts lost **and** next pick/ban lost
    - 20 minutes or more: **Forfeit** unless the opponent wants to reschedule
19. If a player becomes silenced while playing a map, it counts as a disconnect.

### Schedule Rules

1. Qualifiers lobbies should be scheduled at least **6 hours** in advance.
2. In qualifiers, players may schedule an EX lobby at any time. Depending on referee availability, this may not be granted.
3. Matches should be scheduled at least **24 hours** in advance.
4. Reschedule requests should be considered reasonably by the receiving player. If a player appears to be denying reschedule requests unfairly, the administration will intervene.
5. Reschedule requests with faked confirmation from the other player will result in a forfeit.

### Group Stage

1. During the Group Stage, there are 3 to 5 players in each group, depending on registrations.

> 2. These players all play 1 match against each other over two weeks.

Reviewer: bit weird to mix these in the same sentence

pedantically, small numbers in sentences should *either* be spelled out or written in numerals, for the whole article

```suggestion
2. These players all play one match against each other over two weeks.
```

3. The placements are determined by the following criteria, in the order they are applied:
   - Amount of match wins
   - Maps won minus maps lost

>    - Direct comparison of a pair's group stage match (e.g. P1 vs P2: 5-1, so P1 gets the better placement)

Reviewer: ```suggestion
   - Direct comparison of a pair's Group Stage match (e.g. if player 1 wins 5–1 against player 2, player 1 gets the better placement)
```

### Screening

1. Players can be screened for having performed **well above** the level of the rank range or for having suspicious profile stats. The following guidelines (not strict criteria, appealable) are used to determine this:

>    - osu!standard tournament badge(s)
>    - Having a play worth over 300pp

Reviewer: "osu!standard" isn't the official term, it's just "osu!"

```suggestion
   - Tournament badges in the osu! game mode
   - Having a play worth over 300 pp
```

   - Suspiciously low playtime
   - Suspiciously recent account registration date
   - Rank <95k during registration phase

>    - Rank <95k after adding tournament top plays (mixed pp) and unranked osu!lazer plays

Reviewer: ```suggestion
   - Rank <95k after adding tournament top plays (mixed pp) and unranked osu!(lazer) plays
```

   - Suspicious profile activity
   - High average scores on pools above 5.5 stars

>    - Consistent high performer in major 100k tournaments (e.g. EQX, 6EC24, GC#1, 6WC23)

Reviewer: would you mind linking these tournaments? it's very hard to search by acronym in most cases

Author: Fixed.

   - Trying to mislead the screening process (for example, by faking match history)
   - Other tournament overperformance
2. It is possible to appeal a screening if you think it was unjustified or a mistake. To do this, message either host (preferably through Discord) and explain why you believe your screening was unjustified.
3. Screenings should be appealed **as soon as possible** to have enough time to play Qualifiers.
4. Players cannot be screened for bad reputation from before the tournament.

### Self-ref

1. Players may self-ref their matches if necessary. Permission to self-ref is granted by the administration.
2. Self-ref requests will be granted only under all of the following conditions:
   - There are no referees available
   - The match is during Group Stage
   - The match is not streamed

> 3. If you self-ref a match, post the MP link on the Discord server at the start of the match.

Reviewer: ```suggestion
3. If you self-ref a match, post the match history link in the Discord server at the start of the match.
```

4. The tournament administration will inform you if the match was valid or invalid after it completes.

> 5. There will only be asyncs if absolutely necessary. Even if permission is granted, taking an async results in you losing your warmups, protects, and bans in your next match.

Reviewer: better to spell this one out since it's a rather obscure term

```suggestion
5. Asynchronous matches are only allowed if absolutely necessary. Even if permission is granted, playing a match asynchronously results in you losing your warmups, protects, and bans in your next match.
```
